Android.mk

```diff
@@ -228,6 +229,8 @@ $(TARGET_BLOBS_MAP): $(TARGET_BLOBS_SHA1_OUT)
 $(TARGET_CMDLINE_FS):
 	echo "system.img: -s -l $(BOARD_SYSTEMIMAGE_PARTITION_SIZE) -a system" > $@
 	echo "userdata.img: -s -l $(BOARD_USERDATAIMAGE_PARTITION_SIZE) -a userdata" >> $@
+	echo "boot.img: $(BOARD_MKBOOTIMG_ARGS)" >> $@
+	echo "recovery.img: $(BOARD_MKBOOTIMG_ARGS)" >> $@
```

Comment:
Nice, I just checked and this is something we will need for Sony Shinano/Yukon platform-based devices using the AOSP L port :)